k8s/overlays/azure/common/kustomization.yaml

```diff
@@ -4,8 +4,10 @@ kind: Kustomization
 resources:
   - ../../../bases/common
   - azure-storage-class.yaml
+  - azure-storage-azure-disk.yaml
   - minio-pv-claim.yaml
   - postgresql-pv-claim.yaml
+  - postgresql-pv-claim-azure-disk.yaml
```

Comment:
Maybe rename it to `postgresql-data-pv-claim` in order to be consistent with the naming elsewhere in the project